libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/dto/VerifiableCredentialResponseDto.java

```diff
@@ -15,6 +16,7 @@
 @Builder
 @Getter
 @Setter
+@Jacksonized
```

Comment:
Curious what this is for - there are a lot of annotations here 😅

Reply:
Ah yep, this is from experimentation - will remove